apps/www/src/lib/registry/default/ui/calendar/Calendar.vue

```diff
@@ -125,7 +125,13 @@ const vCalendarSlots = computed(() => {
   @apply flex;
 }
 .calendar .vc-weekday {
-  @apply text-muted-foreground rounded-md w-9 font-normal text-[0.8rem];
+  @apply text-muted-foreground rounded-md w-full font-normal text-[0.8rem];
+}
+.calendar .vc-weekday-1 {
+	@apply pr-3;
+}
+.calendar .vc-weekday-7 {
+	@apply pl-3;
 }
```

Comment:
But these styles are redundant I think

Can you explain what is these change for? Thanks

Reply:
@sadeghbarati Yeah I'm Boo 😄. I noticed that all `weeks` elements had either an `on-left` or `on-right` class which was applying a left/right padding to the last/first `vc-day`.

I couldn't figure out how to apply that class so I brute forced it with padding.

I've just seen now that it's possible to change the first day of the week... so this change I've made is not good because it won't work for other `first-day-of-week` values.

Behaviour I was going for was to make the weekday letters line up with the day buttons. For example the two S weekdays in your image above are off-center.
![image](https://github.com/radix-vue/shadcn-vue/assets/26427343/fdce1fe2-89f1-4d9c-ba79-9e9b9571fdde)

---
Probably also worth noting that when you increase calendar width the "current day" has that large background color like in your image above. I "fixed" that like below adding the background to the day instead of the container. I didn't include in this PR because wasn't sure if it was intentional behaviour. The difference is also showcased in Before/After image.
![image](https://github.com/radix-vue/shadcn-vue/assets/26427343/064a93a7-49a8-4db5-889b-681996349398)

Reply:
@JacksonBowe Thanks for explaining


Can you please also push your unincluded change for `.v-day` and `vc-day.is-today` 👍 


https://github.com/radix-vue/shadcn-vue/assets/17789047/e385f23c-6fcd-45c0-bfcc-597bb400bbfe